week1/homework/src/server.js

```diff
@@ -10,11 +10,56 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
-  });
 
+    switch (request.url) {
+      case '/state':
+        state = 10;
+        let number = JSON.stringify({
+          state: state
+        });
+        response.setHeader('Content-Type', 'application/json');
```

Comment:
You can also put the `setHeader` outside of the switch so you don't have to repeat it.